style.css

```diff
@@ -0,0 +1,93 @@
+/* GLOBAL RULES */
+
+:root {
+  --main-color: #172b4d;
+  --secondary-color: #36b37e;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  /* font-size: 10px - 10px / 16px =0.625 = 62.5% - percentage of user's browsers font-size setting - 1rem = 10 px */
+  font-size: 62.5%;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  line-height: 1;
+  color: var(--main-color);
+}
+
+ul {
+  list-style: none;
+}
+
+h1 {
+  text-align: center;
+  font-size: 3.2rem;
+  margin: 5.2rem;
+}
+
+.container {
+  max-width: 100rem;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 1.2rem;
+  height: calc(100vh - 24rem);
+}
```

Comment:
try to practice the responsive rules on this grid system because the design didn't work on small screens